nodes/tensor_utils/save_text_tensor.py

```diff
@@ -0,0 +1,26 @@
+import json
+
+from comfystream import tensor_cache
+
+class SaveTextTensor:
```

Comment:
@ad-astra-video, @varshith15 we can probably improve the naming. It can send any data in json format.